### ISSUE TITLE
[BUG] After reparsing CommandContext arguments are empty

### ISSUE THREAD

**Issue by XjCyan1de** [Pante/Chimera#191]
**Describe the bug**
After reparsing CommandContext in SpigotMapper arguments become empty

**To Reproduce**
```java
        var second = Argument.of("second", StringArgumentType.word())
                .suggests((context, builder) -> {
                    context.getArgument("first", String.class); // No such argument 'first' exists on this command
                    return builder.buildFuture();
                });

        var first = Argument.of("first", StringArgumentType.word());

        Literal.of("test").then(first.then(second));
```

**Expected behavior**
context.getArgument("first", String.class) returns true value

**Screenshots**
https://i.imgur.com/dWvrnjo.png

**Environment (please complete the following information):**
 - Java Version - OpenJDK 64-Bit Server VM 14.0.1+7
 - Platform - git-Paper-87
 - Platform Version - 1.16.1
- Chimera Version - 4.7.0

**Comment by Pante** [comment]
Thanks for taking the time to report the bug. After some investigation, it turns out that `CommandDispatcher` does not trim the leading `/` in a given input before parsing it. Seems it re-parsing a `Type<?>` also faces the same issue. 

A fix has been targeted for `4.7.1` which will be released later today, in the interim, please use `4.7.1-SNAPSHOT` in
```xml
<repository>
  <id>chimera-snapshots</id>
  <url>https://repo.karuslabs.com/repository/chimera-snapshots/</url>
</repository>
```

**Comment by Pante** [comment]
Fixed in 4.7.1, please do re-open this issue if (in the unlikely event) the bug reoccurs.
